components/mediation-ui/org.wso2.carbon.inbound.ui/src/main/resources/web/inbound/saveInbound.jsp

```diff
@@ -73,8 +73,10 @@
                     sParams.add((new ParamDTO("SSLProtocol",request.getParameter(strKey))));
                 }else if(strKey.startsWith("CertificateRevocationVerifier")){
                     sParams.add((new ParamDTO("CertificateRevocationVerifier",request.getParameter(strKey))));
+                }else if(strKey.startsWith("enableSSL")){
+                    sParams.add((new ParamDTO("enableSSL",request.getParameter(strKey))));
                 }else if(strKey.startsWith("coordination")){
-		    sParams.add((new ParamDTO("coordination",request.getParameter("coordination"))));
```

Comment:
@prabathariyaratna : Please remove the code changes which are introduced due to formatting.